src/components/generic/Button.js

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components';
+import { ThemeProvider } from 'styled-components';
+
+// Palette generated @ https://coolors.co/
+const theme = {
+  start: {
+    bgColor: "#44AF69",
+    color: "#ffffff"
+  },
+  pause: {
+    bgColor: "#FCAB10",
+    color: "#000000"
+  },
+  resume: {
+    bgColor: "#2B9EB3",
+    color: "#ffffff"
+  },
+  fastForward: {
+    bgColor: "#DBD5B5",
+    color: "#000000"
+  },
+  reset: {
+    bgColor: "#F8333C",
+    color: "#ffffff"
+  }
+};
+
+const StyledButton = styled.button`
+  font-size: 17px;
+  text-transform: uppercase;
+  margin: 5px;
+  padding: 8px 14px;
+  border-radius: 5px;
+  font-weight: 700;
+  border: 0;
+  letter-spacing: 0.5px;
+  cursor: ${props => {if (!props.disabled) return 'pointer';}};
+  opacity: ${props => {if (props.disabled) return '0.4';}};
+  background-color: ${props => {
+    if (props.value === 'Start') return props.theme.start.bgColor;
```

Comment:
I would not use the `value` prop of the button as the key to style it - this is a free form label that a developer can easily change. instead would probably use something like `type` to make your Buttons more generic.